bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LegacyCookieValidation.java

```diff
@@ -94,7 +94,7 @@ public void checkCookies(List<File> directories) throws BookieException {
             if (!missedCookieDirs.isEmpty()) {
                 if (rmCookie == null) {
                     // 5.1 new environment: all directories should be empty
-                    verifyDirsForNewEnvironment(missedCookieDirs);
+                    verifyDirsForNewEnvironment(directories);
```

Comment:
Have you encountered any exceptions? The missedCookieDirs is the result of checking remote cookies and directories